dips/dip-7.md

```diff
@@ -14,40 +14,56 @@ This DIP outlines the process by which technical working groups (TWG’s) are fo
 ## Formation of a Technical Working Group
 Technical Working Groups (TWG) are formed to address or explore specific technical topics or issues as they relate to Diem. TWGs collect information and expert input on a particular topic, discuss material changes to the Diem Payment Network (DPN), and create outputs in the form of recommendations, reports, and Diem Improvement Proposals (DIPs).
 
-The proposal to form a technical working group can be made by any Association member. The proposal must include, but is not limited to:
-The group’s scope and goal
-The limits of its focus
-The expected final output, typically as a DIP, report, or recommendation
-Expected duration of operation
-The initial facilitator (up to two individuals)
-A brief description of ideal desired member profiles so other Association members can join
+The proposal to form a technical working group can be made by anyone and is not limited to current Association members. The proposal must include, but is not limited to:
+- The group’s scope and goal
+- The limits of its focus
+- The expected final output(s), typically as a DIP, report, or recommendation
+- Expected duration of operation (typically 3-6 months)
+- The initial facilitator (up to two individuals)
+- A brief description of ideal desired member profiles so other community or Association members can join
+- Whether the TWG is intended to be public or by invitation (public is default)
 
-Once the proposal is created, the facilitator will circulate it in public slack channels relevant to the topic. The facilitator is also encouraged to proactively share the proposal with Association members that are likely to be impacted by this TWG or that could contribute to it. The facilitator must circulate the proposal over 1-2 weeks for sufficient interest to gather.
+Once the proposal is created, the facilitator will circulate it in public slack channels and other mediums relevant to the topic. The facilitator is also encouraged to proactively share the proposal with Association members that are likely to be impacted by this TWG or that could contribute to it. The facilitator must circulate the proposal over 1-2 weeks for sufficient interest to gather.
 
-After gathering interest, the facilitator must choose members to be part of the TWG, add them to the proposal, and identify their roles (co-facilitator, contributor, reviewer, etc.). The proposal must have representation from a minimum of 3 Association members to build collaboration and as a tool to determine relevance to the entire association, rather than an individual member.
+Anyone can join a TWG by asking for the facilitator(s) to add them to the proposal (and at any time afterwards) and identifying their roles (co-facilitator, contributor, reviewer, etc.). It is recommended for the proposal to have representation from a minimum of 3 Association members to build collaboration and as a tool to determine relevance to the entire association, rather than an individual member.
```

Comment:
Add that a request to join may be denied (and on what considerations -- sensitive topic, insufficient familiarity with requestor, insufficient expertise level, reached size limit, etc.). Diem will have a lot of enemies, fraudsters, and even just curious bystanders.